### ISSUE TITLE
bug: Latest langchain-experimental support

### ISSUE THREAD

**Issue by yamonkjd** [logspace-ai/langflow#1123]
I'm trying to use the latest langchain-experimental, but only version 0.0.8 of langflow is compatible. Currently version 0.0.38 is the latest, so please add support for it.

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
langflow 0.5.3 requires langchain-experimental<0.0.9,>=0.0.8, but you have langchain-experimental 0.0.38 which is incompatible.
```

I tried to implement CSV agent as a custom agent. However, the versions are not compatible in Langflow.

**Comment by yamonkjd** [comment]
We close the issue by confirming that the dependencies have been modified to be compatible with all experimental versions in the latest version.

**Comment by Arron-Clague** [comment]
Hi there : I am also having this issue with CSVAgent ! Was it ever resolved ? Im using 0.6.4 of langflow
